node/runtime/pangoro/src/pallets/to_substrate_backing_.rs

```diff
@@ -7,7 +7,8 @@ use frame_support::PalletId;
 // --- darwinia-network ---
 use crate::{pangolin_messages::ToPangolinOutboundPayload, *};
 use bridge_primitives::{AccountIdConverter, PANGORO_PANGOLIN_LANE};
-use darwinia_support::s2s::LatestMessageNoncer;
+use darwinia_support::{evm::IntoH160, s2s::LatestMessageNoncer};
+use dp_asset::{TokenMetadata, NATIVE_TOKEN_TYPE};
 use to_substrate_backing::Config;
 pub struct PangolinMessageNoncer;
```

Comment:
Can we remove the `MessageNoncer` config in this pallet? Because we don't need to do the message-id conversion.  The `outbound_latest_generated_nonce` and `inbound_latest_received_nonce` can be query via `type MessagesBridge`. Just a suggestion, maybe I missed something important.

Reply:
`MessagesBridge ` have not implement the trait for outbound_latest_generated_nonce or inbound_latest_received_nonce, we can't query them by this type from pallet.